appengine-jetty-managed-runtime/src/main/java/com/google/apphosting/vmruntime/jetty9/VmRuntimeWebAppContext.java

```diff
@@ -262,129 +260,97 @@ public boolean isTrustedRemoteAddr(String remoteAddr) {
     return VmRequestUtils.isTrustedRemoteAddr(isDevMode, remoteAddr);
   }
 
-  /**
-
-   * Overrides doScope from ScopedHandler.
-   *
-   *  Configures a thread local environment before the request is forwarded on to be handled by the
-   * SessionHandler, SecurityHandler, and ServletHandler in turn. The environment is required for
-   * AppEngine APIs to function. A request specific environment is required since some information
-   * is encoded in request headers on the request (for example current user).
-   */
-  @Override
-  public final void doScope(
```

Comment:
@ludoch Note that we no longer override doScope.   It should be possible to move all of this behaviour into listeners, which will make the maintenance easier.